contents/blog/what-is-a-growth-engineer.md

```diff
@@ -0,0 +1,108 @@
+
+Last is **data literacy**. Since growth engineers do a lot of experiments, they must understand how those experiments perform. They should know what factors are relevant, how to calculate if something is significant, and what data to care about in the first place. All of this requires at least a basic knowledge of data and statistics.
+
+![Growth engineers' power](../images/blog/what-is-a-growth-engineer/thanos.jpg)
+
+If this sounds like a unicorn, you aren’t wrong. This is again why we find former technical founders as a good fit for this role. They have the business and product knowledge to develop growth ideas and strategies, the practical technical knowledge to implement them, as well as a familiarity with the data a product produces.  
+
+## What do growth engineers actually work on?
+
+So what do growth engineers use their skills to do? We’ve already covered that they work to increase signups, subscriptions, and revenue by working across the product, but what are some examples of what this looks like?
+
+- Developing a growth strategy, experiment framework, and measurement for both.
+- Doing SEO optimization on the marketing site. Building landing pages. Optimizing the signup funnels.
+- Running A/B tests on specific parts of the signup and activation funnel. Trying to improve the conversion rate for both.
+- Setting up in-app and push notification messaging systems. Building tools and systems for better product onboarding.
+
+### What do PostHog’s growth engineers work on?
+
+To give a specific example of what growth teams work on, we can look at our [growth team at PostHog](/handbook/small-teams/growth). Over the past few months, this team focused on three areas:
+
+1. **Billing.** Creating a [billing service](https://github.com/PostHog/posthog/pull/11986) enabling different pricing for different plans and [products](https://github.com/PostHog/posthog/pull/15035/files). Improving the billing page both on the [website](https://github.com/PostHog/posthog.com/pull/5181) and [in-app](https://github.com/PostHog/posthog/pull/14652) to work with this service.
+2. **Signup.** Improving the signup funnel. Running A/B tests of options like [single sign on buttons](https://github.com/PostHog/posthog/pull/13234). [Verifying emails](https://github.com/PostHog/posthog/pull/13632) and adding two-factor authentication.
+3. **Onboarding.** Setting up [in-app messaging](https://github.com/PostHog/posthog/pull/13108) and notifications. Set up customer.io. Add [autocapture](https://github.com/PostHog/posthog/pull/14687) and session recording toggles.
```

Comment:
I really like that these to Pull Requests! makes it interesting to snoop around in